packages/freebasic.rb

```diff
@@ -1,16 +1,17 @@
 require 'package'
 
 class Freebasic < Package
-  description 'FreeBASIC is a free/open source (GPL), BASIC compiler for Microsoft Windows, DOS and Linux.'
+  description 'FreeBASIC is a free and open source, BASIC compiler for Microsoft Windows, DOS and Linux.'
   homepage 'https://www.freebasic.net/'
   version '1.07.1-1'
+  license 'LGPL-2 and GPL-2'
   compatibility 'i686,x86_64'
   case ARCH
   when 'i686'
-    source_url 'https://downloads.sourceforge.net/project/fbc/Binaries%20-%20Linux/FreeBASIC-1.07.1-linux-x86.tar.gz'
+    source_url 'https://downloads.sourceforge.net/project/fbc/Binaries%20-%20Linux/FreeBASIC-1.07.1-linux-x86.tar.gz' # 404 not found
```

Comment:
New version at https://sourceforge.net/projects/fbc/files/Binaries%20-%20Linux/FreeBASIC-1.07.2-linux-x86.tar.gz?

Reply:
Do you think we shud get this pr merged first and then fix that? New versions requires testing

Reply:
Yes, makes sense to include in a separate PR.